src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1100,18 +1100,18 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                                                 $offsetInLine = 0
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-macos`
ConsoleHost unit tests.Pipe to/from powershell.errors are in text if error is redirected, encoded command, non-interactive, and outputformat specified
```powershell
Expected strings to be the same, but they were different.
Expected length: 4
Actual length:   21
Strings differ at index 4.
Expected: 'boom'
But was:  'boomAt line:1 char:25'
at <ScriptBlock>, /Users/runner/runners/2.164.6/work/1/s/test/powershell/Host/ConsoleHost.Tests.ps1: line 417
417:             $out.Split([Environment]::NewLine)[0] | Should -BeExactly "boom"
```
Write-Error Tests.ErrorRecord should not be truncated or have inserted newlines when redirected from another process
```powershell
Expected exactly 3, but got 1.
at <ScriptBlock>, /Users/runner/runners/2.164.6/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Error.Tests.ps1: line 105
105:         $result.Count | Should -BeExactly 3
```